.github/workflows/run-tests.yml

```diff
@@ -231,3 +228,77 @@ jobs:
 #          comment-id: ${{ steps.process-vars.outputs.COMMENT_ID }}
 #          edit-mode: append
 #          body: ${{ steps.process-vars.outputs.COMMENT_BODY }}
+
+  run-tests:
+    name: Run tests
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      GH_WORKSPACE: ${{ github.workspace }} # used in docker compose
+      DEFAULT_UBUNTU_BASE: '20.04'
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # tests run on 20.04 (Qt 5.12), compile test on 20.10 (Qt 5.14)
+        ubuntu-base: [20.04]
+        test-batch: [ALL_BUT_PROVIDERS, POSTGRES, ORACLE, HANA]
+      fail-fast: false
+
+    needs: build
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set vars
+        env:
+          GITHUB_EVENT_NAME: ${{ github.event_name }}
+          GITHUB_REF: ${{ github.ref }}
+          GITHUB_PR_NUMBER: ${{github.event.number}}
+          UBUNTU_BASE: ${{ matrix.ubuntu-base }}
+          TEST_BATCH: ${{ matrix.test-batch }}
+        run: |
+          DOCKER_TAG=$(echo $( [[ ${GITHUB_EVENT_NAME} =~ ^pull_request$ ]] && echo ${GITHUB_BASE_REF} || echo ${GITHUB_REF##*/} ) | sed 's/^master$/latest/')$( [[ ${UBUNTU_BASE} != ${DEFAULT_UBUNTU_BASE} ]] && echo "_${UBUNTU_BASE}" || echo "" )
```

Comment:
could you add a remark here that is duplicate code from above and that both need to be updated (put the remark at the 2 places)

Reply:
Done